LICENSE.md

```diff
@@ -0,0 +1,13 @@
+Copyright 2018 redhat
```

Comment:
I think that this is intended to be embedded in other files, and that we should also include the full unmodified text of the license in a `LICENSE` file, at least that is what other projects do. For example: https://github.com/apache/httpd/blob/trunk/LICENSE